.gitignore

```diff
@@ -13,6 +13,7 @@ project/boot/
 project/plugins/project/
 patch-process/*
 .idea
+!/.idea/codeStyles
```

Comment:
the whole `idea` folder is not in version control, so do we need this line?

Reply:
> the whole `idea` folder is not in version control, so do we need this line?

It has to be, and it's very important.

This file will allow you to import your idea automatically into the project's code format

Reply:
It seems to me using build tool to address that is more acceptable. We all use gradle to build kafka but there are many IDE.

We can add IDE supports in the future but could you please keep this PR simple for now?